packages/browser/__tests__/Session.spec.ts

```diff
@@ -271,37 +297,33 @@ describe("Session", () => {
       expect(mySession.info.sessionId).toEqual("mySession");
     });
 
-    function sleep(ms: number): Promise<void> {
-      return new Promise((resolve) => setTimeout(resolve, ms));
```

Comment:
Since I had to mock `setTimeout` for the other test I couldn't use this improvised `sleep` function anymore, but I think I've rewritten it to the same effect. Specifically, rather than just retrying every `ms` and checking whether `secondRequestIssued` was `true` already, I just manually resolve the Promise from the first request _after_ I issued the second one.